### PR TITLE
W-12185228 mask auth headers bsj

### PR DESCRIPTION
**Pull request by bjoson-mulesoft**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/_partials/afm-task-config-secrets.adoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
+// tag::config-in-secrets-mgr[]
+
+[[configure-secret]]
+= Configure a Secret in Anypoint Secret Manager
+
+To keep sensitive information secret in tests you run from a private location, create a shared secret as a symmetric key in Secrets Manager. If the sensitive information that you want to store as a shared secret is currently in plain text, encode it in Base64 before you create the secret.
+
+To configure a shared secret:
+
+. Open a secrets group or create a new one. Create the group in the same environment that you are using in the BAT CLI.
```
Comment (dhirsch1) [inline modules/ROOT/pages/_partials/afm-task-config-secrets.adoc:10]: Maybe - ...in the same environment that the BAT CLI uses.

File: modules/ROOT/pages/_partials/afm-task-config-secrets.adoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+[[configure-secret]]
+= Configure a Secret in Anypoint Secret Manager
+
+To keep sensitive information secret in tests you run from a private location, create a shared secret as a symmetric key in Secrets Manager. If the sensitive information that you want to store as a shared secret is currently in plain text, encode it in Base64 before you create the secret.
+
+To configure a shared secret:
+
+. Open a secrets group or create a new one. Create the group in the same environment that you are using in the BAT CLI. 
++
+To find out which environment you are currently using in the BAT CLI:
```
Comment (dhirsch1) [inline modules/ROOT/pages/_partials/afm-task-config-secrets.adoc:12]: To find out which environment the BAT CLI uses:

File: modules/ROOT/pages/_partials/afm-task-config-secrets.adoc [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+. Copy the name of the new shared secret obtained in <<configure-secret>>.
+. At a command prompt, run the `bat grant` command, specifying an alias for the shared secret. When you run this command, the BAT CLI creates a section named `secrets` in your test suite's `bat.yaml` file, if the section does not already exist. In that section, the BAT CLI adds these indented lines:
++
+----
+alias:
+ secretId: "secret-ID"
+----
++
+* `alias`: This is the alias that you specified in the `bat grant` command.
```
Comment (dhirsch1) [inline modules/ROOT/pages/_partials/afm-task-config-secrets.adoc]:
```suggestion
* `alias`: The alias that you specified in the `bat grant` command.
```

File: modules/ROOT/pages/_partials/afm-task-config-secrets.adoc [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+* `secret-ID`: This is the ID of the secret within Anypoint Secrets Manager. This ID does not appear in ASM, so there is no way for someone looking in your `bat.yaml` file to associate the ID with any particular secret. The BAT CLI uses this ID to look up the secret that you associated with the alias.
```
Comment (dhirsch1) [inline modules/ROOT/pages/_partials/afm-task-config-secrets.adoc]:
```suggestion
* `secret-ID`: The ID of the secret within Anypoint Secrets Manager. This ID does not appear in ASM, so there is no way for someone looking in your `bat.yaml` file to associate the ID with any particular secret. The BAT CLI uses this ID to look up the secret that you associated with the alias.
```

File: modules/ROOT/pages/afm-mask-info-in-monitors.adoc [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+[[add-secret-to-monitor]]
+== Add the Secret to Your Monitor
+
+Modify your monitor to use the information you configured in Secrets Manager.
+
+To add the secret to a monitor:
+
+. Create or download a monitor. See xref:afm-create-monitor.adoc[Creating Monitors] or xref:afm-download-test.adoc[Downloading Monitors].
+. Modify the `main.dwl` file to replace the sensitive information with the client ID and client secret you configured in Secrets Manager. In the following example, the sensitive information in headers are masked. The 'clientId-alias' and 'clientSecret-alias' are aliases for the secrets stored in Secrets Manager. See <<grant-access-to-secret>> to learn how to generate the alias.
```
Comment (dhirsch1) [inline modules/ROOT/pages/afm-mask-info-in-monitors.adoc]: Suggestion: Put "In the following example" on a new line so it stands out more.

File: modules/ROOT/pages/bat-bdd-reference.adoc [hunk 6]
```diff
@@ -85,32 +85,29 @@ A test suite must contain at least one test (.dwl) file.
...
 In your tests, you can access shared secrets that are symmetric key types from Anypoint Secrets Manager. Secrets Manager enables you to securely store sensitive information, such as a password, authentication token, endpoint URL, or webhook URL.
 
-[IMPORTANT]
-====
-Functional Monitoring only works with symmetric key shared secret types. Using other types from Anypoint Secrets Monitor will cause all used shared secrets to stop being populated.
-====
+=== Limitations
 
-You create aliases for your shared secrets with the `bat grant` command. The BAT CLI stores aliases in your `bat.yaml` files in your test suites. You place aliases in your tests and, at runtime, the BAT CLI looks up the shared secrets by means of the aliases.
+* Functional Monitoring works only with symmetric key shared secret types. Using other types from Anypoint Secrets Monitor will cause all used shared secrets to stop being populated.
```
Comment (dhirsch1) [inline modules/ROOT/pages/bat-bdd-reference.adoc]:
```suggestion
* Functional Monitoring works only with symmetric key shared secret types. Using other types from Anypoint Secrets Monitor causes all used shared secrets to stop being populated.
```

**Review by dhirsch1 — DISMISSED** [review]

File: modules/ROOT/pages/afm-mask-info-in-monitors.adoc (hunk 5 above)
Comment (bjoson-mulesoft) [reply modules/ROOT/pages/afm-mask-info-in-monitors.adoc]: Good call

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**